node/include/cocaine/api/stream.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <cocaine/hpack/header.hpp>
```

Comment:
1) we pass by value, so we do require definition
2) everybody who uses this interface must be aware of hpack::header_storage_t, as all stream_t methods require it.